plugins/Kaleidoscope-EEPROM-Settings/src/kaleidoscope/plugin/EEPROM-Settings.cpp

```diff
@@ -164,9 +164,17 @@ EventHandlerResult FocusSettingsCommand::onFocusEvent(const char *command) {
     GET_CRC,
   } sub_command;
 
-  if (::Focus.handleHelp(command, PSTR("settings.defaultLayer\r\nsettings.valid?\r\nsettings.version\r\nsettings.crc")))
+
+  const char *focus_command_settings_      = PSTR("settings");
+  const char *focus_command_default_layer_ = PSTR("defaultLayer");
+  const char *focus_command_is_valid_      = PSTR("valid?");
+  const char *focus_command_version_       = PSTR("version");
+  const char *focus_command_crc_           = PSTR("crc");
+
+  if (::Focus.handleHelp(command, 11, focus_command_settings_, focus_command_default_layer_, Focus.CRLF, focus_command_settings_, focus_command_is_valid_, Focus.CRLF, focus_command_settings_, focus_command_version_, Focus.CRLF, focus_command_settings_, focus_command_crc_))
     return EventHandlerResult::OK;
 
+
   if (strncmp_P(command, PSTR("settings."), 9) != 0)
```

Comment:
>Talking about patterns, one of the patterns currently used, one that this refactor does not change at all, is [this](https://github.com/keyboardio/Kaleidoscope/blob/961853e3f30a9f02072a3c49af3fab64b8cfc7f4/plugins/Kaleidoscope-EEPROM-Settings/src/kaleidoscope/plugin/EEPROM-Settings.cpp#L178-L190):

(I had failed to finish refactoring EEPROM-Settings to use the new comparators in the if statement branches.)

Reply:
> (I had failed to finish refactoring EEPROM-Settings to use the new comparators in the if statement branches.)

I know, but even with the new helpers, the pattern would be the same. It would have less magic numbers, but otherwise it would flow exactly the same. Apologies if my comment wasn't clear.

Reply:
Sure. The goal here was something we'd discussed in #kaleidoscope-dev - an initial refactoring primarily for clarity and extracting some repeated patterns, rather than a significant retool of control flow.